pkg/security/filters/required_authz.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package filters
+
+import (
+	"github.com/Peripli/service-manager/pkg/log"
+	"github.com/Peripli/service-manager/pkg/security"
+	"github.com/Peripli/service-manager/pkg/web"
+)
+
+// RequiredAuthorizationFilterName is the name of RequiredAuthorizationFilter
+const RequiredAuthorizationFilterName = "RequiredAuthorizationFilter"
+
+// requiredAuthzFilter type verifies that authorization has been performed for APIs that are secured
+type requiredAuthzFilter struct {
```

Comment:
`type requiredAuthzFilter struct {}`

instead of:

```
type requiredAuthzFilter struct {
}
```
for the sake of consistency with **required_authn.go**